factcast-schema-registry-cli/src/main/resources/site/hugo-theme-docdock-master/README.md

```diff
@@ -0,0 +1,129 @@
+# Hugo docDock Theme
+
+This repository contains a theme for [Hugo](https://gohugo.io/), based on 
+
+* [Matcornic Learn theme](https://github.com/matcornic/hugo-theme-learn/).
+* [facette.io](https://facette.io/)'s documentation style css (Facette is a great time series data visualization software)
+
+Visit the [theme documentation](http://docdock.netlify.com/) to see what is going on. It is actually built with this theme.
+
+# Main features
+
+- Search
+- **Unlimited menu levels**
+- RevealJS presentation from markdown (embededed or fullscreen page)
+- Attachments files
+- List child pages
+- Include segment of content from one page in another (Excerpt)
+- Automatic next/prev buttons to navigate through menu entries
+- Mermaid diagram
+- Icons, Buttons, Alerts, Panels, Tip/Note/Info/Warning boxes
+- Image resizing, shadow...
+- Customizable look and feel
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4955738)